tests/e2e/specs/admin-tracking.test.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * WordPress dependencies
+ */
+import { activatePlugin, deactivatePlugin, visitAdminPage } from '@wordpress/e2e-test-utils';
+import { URL } from 'url';
+import { resetSiteKit } from '../utils';
+
+describe( 'Providing client configuration', () => {
+
+	beforeAll( async() => {
+		await activatePlugin( 'google-site-kit' );
+		await activatePlugin( 'e2e-tests-auth-plugin' );
+		await resetSiteKit();
+	} );
+
+	beforeEach( async() => {
+		await visitAdminPage( 'admin.php', 'page=googlesitekit-settings' );
+		await page.waitForSelector( '.mdc-tab-scroller__scroll-content' );
+		page.click( '.mdc-tab-scroller__scroll-content button:last-child' );
+	} );
+
+	afterAll( async() => {
+		await resetSiteKit();
+		await deactivatePlugin( 'google-site-kit' );
+		await deactivatePlugin( 'e2e-tests-auth-plugin' );
+	} );
+
+	it( 'Should select opt-in by default', async() => {
+
+		await page.waitForSelector( '#opt-in' );
+
+		expect( await page.$eval( '#opt-in', el => el.checked ) ).toBe( true );
```

Comment:
We currently successfully use `toMatchElement` to make these assertions simpler to read and write. Example here:

```js
await expect( page ).toMatchElement( '#opt-in[checked]' );
```

Reply:
That matches the html of the element when it was rendered on page load, not the current state.

If the toggle was enabled when the page loaded, it will have a `checked` attribute, but otherwise it will not. This attribute does not change when the input is toggled, that's why we're checking the property on the element.

Reply:
Ah, yes of course. Thanks for clarification :-)

Reply:
Updated a couple that could be simplified when the checkbox did not toggle.